library/src/main/java/com/google/android/exoplayer2/text/ImageCue.java

```diff
@@ -0,0 +1,41 @@
+package com.google.android.exoplayer2.text;
+
+import android.graphics.Bitmap;
+
+public class ImageCue extends Cue {
+
+  final private long start_display_time;
+  final private int x;
+  final private int y;
+  final private int bitmap_height;
+  final private int bitmap_width;
+  final private int plane_height;
+  final private int plane_width;
+  final private Bitmap bitmap;
+  final private boolean isForced;
```

Comment:
We already support forcing at the Format level (Format.selectionFlags can include C.SELECTION_FLAG_FORCED).